### DISCUSSION TITLE
Simple Page from Docs

### DISCUSSION THREAD

**Discussion by ludwikc** [oleeskild/obsidian-digital-garden#287] [Q&A] [answered]
Hi @oleeskild, 

in the Docs, you've shared this Simple Page, with navigation, graph and other bells and whistles turned off. 
https://dg-docs.ole.dev/example-pages/simple-page/ 

What's the frontmatter tag to use to have it displayed like that regardless of the default Garden configuration?

**Comment by oleeskild** [comment, marked as answer]
Hi 👋 
```
---
dg-publish: true
dg-show-local-graph: false
dg-enable-search: false
dg-show-backlinks: false
dg-show-file-tree: false
dg-home-link: false
dg-show-toc: false
---
```
